src/providers/socketio/socketio.gateway.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  WebSocketGateway,
+  WebSocketServer,
+  OnGatewayConnection,
+  OnGatewayDisconnect,
+  SubscribeMessage,
+  MessageBody,
+} from '@nestjs/websockets';
+import { Socket, Server } from 'socket.io';
+import { Logger } from '@nestjs/common';
+@WebSocketGateway()
```

Comment:
Da um espaço entre o ultimo import e a classe
```suggestion
import { Logger } from '@nestjs/common';

@WebSocketGateway()
```

Reply:
ok